packages/webpack-loader/src/__fixtures__/local-styles.tsx

```diff
@@ -1,13 +1,14 @@
-import { styled } from '@compiled/react';
-import React from 'react';
+/** @jsx jsx */
+import { jsx, styled } from '@compiled/react';
+import { Fragment } from 'react';
 
 const Styled = styled.div({
   color: 'blue',
 });
 
 export const App = (): JSX.Element => (
-  <>
+  <Fragment>
```

Comment:
Does `<>` not work with the `jsx`?

Reply:
Yeah, exactly. Unless you specify the fragment pragma.